### ISSUE TITLE
Removal of grouping by Sheet No.

### ISSUE THREAD

**Issue by amplitudesolutions** [amplitudesolutions/WarehouseRFI#11]
Assume following.

Line No:               127001
Sheets:                 127001-S01
                                127001-S02
                                127001-S03

We generate following:

One Supports Req containing
                All supports from 127001-S01
                All supports from 127001-S02
                All supports from 127001-S03

One Loose piping Items Req containing
                All Loose Piping Items from 127001-S01
                All Loose Piping Items from 127001-S02
All Loose Piping Items from 127001-S03

One Spool Req containing
                All spools from 127001-S01
                All spools from 127001-S02
                All spools from 127001-S03

In other words, we merge all sheets into one requisition.

**Comment by amplitudesolutions** [comment]
Git Branch: RemoveSheetNoGrouping
